CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.7.0
+
+- BREAKING CHANGE: Update all Database types to use a `CommonDatabase` interface.
+- BREAKING CHANGE: Update the openDB and open mthods to be synchronous.
```

Comment:
I think the synchronous methods are not a breaking change, it was like that in the last stable release. Also there is a typo in `mthods` :)